gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/InitialImageOperation.java

```diff
@@ -232,6 +232,16 @@ private GIIStatus reportGIIStatus() {
     }
   }
 
+  private boolean hasLockedBucket = false;
+
+  private void unlockTargetBucketLock(InternalDistributedMember recipient) {
+    if (region instanceof BucketRegion && hasLockedBucket) {
+      this.region.getDistributionManager().getLoggerI18n().info(LocalizedStrings.DEBUG, "Processing SnapshotBucketLockReleaseProcessor to "+ recipient);
+      SnapshotBucketLockReleaseMessage.send(recipient, this.region.getDistributionManager(), region.getFullPath());
```

Comment:
What will happen if the sending of this message fails or/and the reply is not got. In both this case how are we covered that gii will still continue? Looked at the code but it looks like GII will just fail.